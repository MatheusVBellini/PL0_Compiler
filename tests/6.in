PROCEDURE increment;
VAR num;
BEGIN
  num := num + 1;
END.
