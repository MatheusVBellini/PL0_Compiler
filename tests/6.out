1:7 error: Invalid identifier
1:8 error: Expected a period at the end of the program

Compilation failed, 2 errors
